stream-tests/src/test/scala/org/apache/pekko/stream/scaladsl/FlowCollectWhileSpec.scala

```diff
@@ -61,7 +61,7 @@ class FlowCollectWhileSpec extends StreamSpec with ScriptedTest {
 
     "restart when pf throws" in {
       Source(1 to 6)
-        .collect { case x: Int => if (x % 2 == 0) throw TE("") else x }
+        .collectWhile { case x: Int => if (x % 2 == 0) throw TE("") else x }
```

Comment:
can you add some message into exception generate like other Spec does?

Reply:
ok, I will update later, currently i will run the bench :)

Reply:
this is not a typo - a typo is a mispelling. This seems like a bug in the test. Could you correct the name of the PR?